packages/rancher-prom2teams/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,5 +1,11 @@
++annotations:
++  catalog.cattle.io/hidden: "true"
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/namespace: cattle-monitoring-system
```

Comment:
I will keep the first but remove the last one `catalog.rancher.io/namespace: cattle-monitoring-system` because 
- this chart should be hidden in the UI
- and because the UI dose not show this chart, setting the default namespace for it is unnecessary.